src/aircloak/JsonApi/ResponseTypes/Values/AircloakValueJsonParser.cs

```diff
@@ -101,18 +101,18 @@ reader.TokenType switch
         /// </item>
         /// </list>
         /// </exception>
-        private static AircloakColumn<T> ParseColumn<T>(
+        private static AircloakValue<T> ParseColumn<T>(
```

Comment:
Following the general renaming pattern - should this method also be renamed from `ParseColumn` to `ParseValue`?